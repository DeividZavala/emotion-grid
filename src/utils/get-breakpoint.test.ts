import { getBreakpoint } from './get-breakpoint';

describe('Get breakpoint', () => {
  it('should return the current breakpoint', () => {
    const result = getBreakpoint({
      xs: 1,
      sm: 36,
      md: 48,
      lg: 62,
      xl: 75,
    } as any);

    expect(result).toMatchSnapshot();
  });
});
