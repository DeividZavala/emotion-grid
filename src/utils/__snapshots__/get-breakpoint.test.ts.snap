// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Get breakpoint should return the current breakpoint 1`] = `"xl"`;
